### ISSUE TITLE
Message list lines & tags Colors

### ISSUE THREAD

**Issue by FerSabGar** [spymastermatt/thunderbird-monterail#27]
Firstly, many thanks to @spymastermatt for their excellent work, and also for other contributors.
Could it be possible to write lines from 'messages list' according to colors associated to tags? So, could 'important' messages be listed in red, and so on?

**Comment by spymastermatt** [comment]
@FerSabGar Hmm. I'll look into it. It might just be a case of removing the !important flag from the Highlight color rules, but I'm not sure if that will stop them from working all together. I'll have a look tomorrow hopefully

**Comment by FerSabGar** [comment]
It seems that you are right; just deleting '!important' from "#threadTree > treechildren::-moz-tree-cell-text {" structure in _base.css file the colors appears.
So, the resulting code in _base.css file is:
```
#threadTree > treechildren::-moz-tree-cell-text {
	color: var(--message-list-text-color); /* without "!important" makes possible print lines with tags colors */
```
I will test if this change implies additional problems.
Thanks.

**Comment by qazip** [comment]
Changing what @FerSabGar suggested makes it impossible to see message names in the message list, unless a message is selected or tagged. This is true for the full-dark theme, atleast.

**Comment by conema** [comment]
@qazip This issue should be resolved with #40
